druid/src/widget/flex.rs

```diff
@@ -340,9 +340,15 @@ impl FlexParams {
     /// By default, the widget uses the alignment of its parent [`Flex`] container.
     ///
     ///
-    /// [`Flex`]: struct.Flex.html
-    /// [`CrossAxisAlignment`]: enum.CrossAxisAlignment.html
+    /// [`Flex`]: Flex
+    /// [`CrossAxisAlignment`]: CrossAxisAlignment
```

Comment:
```suggestion
```
We can just get rid of this completely and it should work.

Reply:
oooohhhh ok. I did not know this.